patches/api/0001-Wyvern-API-POM.patch

```diff
@@ -0,0 +1,66 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: WyvernPJ <wyvernpj@gmail.com>
+Date: Tue, 8 Feb 2022 19:22:49 +0700
+Subject: [PATCH] Wyvern-API POM
+
+
+diff --git a/pom.xml b/pom.xml
+new file mode 100644
+index 0000000000000000000000000000000000000000..cff111fde321a2aa0966123abdff1a164671f58c
+--- /dev/null
++++ b/pom.xml
+@@ -0,0 +1,54 @@
++<?xml version="1.0" encoding="UTF-8"?>
++<project xmlns="http://maven.apache.org/POM/4.0.0"
++         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
++         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
++    <modelVersion>4.0.0</modelVersion>
++
++    <groupId>com.wyvern</groupId>
```

Comment:
Please revert back to tech.rackdevelopment.wyvern for the groupID, I am only pointing this one out but I am aware there are other files that contain it as well.